src/services/onboard.ts

```diff
@@ -41,12 +40,18 @@ export const createOnboard = (chainConfigs: ChainInfo[], rpcConfig: EnvState['rp
       desktop: { enabled: false },
     },
 
+    notify: {
+      enabled: false,
+    },
+
     appMetadata: {
       name: 'Safe{Wallet}',
-      icon: '/images/safe-logo-green.png',
+      icon: '<svg><image href="/images/safe-logo-green.png" width="56" height="56" /></svg>',
       description: 'Please select a wallet to connect to Safe{Wallet}',
       recommendedInjectedWallets: getRecommendedInjectedWallets(),
     },
+
+    // TODO: Investigate using `autoConnectLastWallet` instead of our `lastWalletStorage`
```

Comment:
❗ 👍

Reply:
I've created [an issue](https://github.com/safe-global/safe-wallet-web/issues/2042) for this to keep track of it better.